### DISCUSSION TITLE
Transferring Beta Projects to a new organization & repo

### DISCUSSION THREAD

**Discussion by sophiatareen** [community/community#12908] [Projects and Issues]
Hello Github Team! We are loving the new Beta Projects. We would like to **transfer our beta project to a new organization and new repo.** Is this possible? What is the best way to accomplish this? Thank you!

**Comment by RyKilleen** [comment]
I also share this desire! Just transferred a personal project to an org, and I'd love to not have to reproduce the whole Beta Project.

**Comment by GMNGeoffrey** [comment]
I put together a hacky script for migrating my team's project. It's pretty bespoke, but might be a useful jumping-off point for someone:

https://gist.github.com/GMNGeoffrey/1768f49339655d2a8ef5f152ab64b0d4

**Reply by AtomicSponge** [reply]
Thank you!  I don't have a huge amount of data to move, but I'd like to preserve it none the less.  Might give this a go as I wanna get more under the hood with github as well

**Comment by GMNGeoffrey** [comment]
Note that there appears to now be functionality to copy a project to another organization, but only the configuration like views and custom fields. This helps a lot though

![image](https://user-images.githubusercontent.com/5732088/214983737-4d14ea8d-8071-417b-be00-6336bc264d74.png)

It would be really nice if GitHub folks could update discussions like this when features are added. These discussion forums feel a lot like shouting into the void sometimes :-/

**Reply by MSIH** [reply]
does this really let you create the project in another org or just within the org?

**Reply by GMNGeoffrey** [reply]
It lets you copy the project configuration to another org, provided you have the permissions to do so (in that screenshot, the original project is in a different org and I am copying it into my personal account)

**Comment by jennijuju** [comment]
Bump this request! the copy feature requires users to migrate issues thats already tracked by the project manually while ideally it could be done automatically.

**Comment by LCK-Locke** [comment]
Would be nice to have a feature like this

**Comment by pombredanne** [comment]
@jennijuju @LCK-Locke See @keshav-space transporter script to copy issues and items https://gist.github.com/keshav-space/f0afa93cb6ec1b9a39299deef9d6858b

**Comment by iyehah** [comment]


**Reply by BlackBeltPanda** [reply]
Reads like a copy/paste from ChatGPT.

**Comment by peteringram0** [comment]
2 years ad still cant do this :(
